### ISSUE TITLE
Z-Wave Migration Doc missing information

### ISSUE THREAD

**Issue by mmiller7** [home-assistant/home-assistant.github.io#21764]
### Feedback

The migration states "Write down/copy your Z-Wave network key somewhere, you are going to need it later."

I don't see this listed anywhere in the old integration or my configuration.yaml.  The directions do not state how or where I should look to figure out what this mysterious "key".

It also does not provide an example or details about what the format of such a key might be to try and recognize it even if I was looking at it.

The documentation should state what the user is supposed to be looking for, and where the user should look to find it.

### URL

https://www.home-assistant.io/integrations/zwave_js/

### Version

2022.2.8

**Comment by MartinHjelmare** [comment]
Here's the documentation for the network key in the old zwave integration:
https://www.home-assistant.io/integrations/zwave/#network-key

**Comment by mmiller7** [comment]
> Here's the documentation for the network key in the old zwave integration: https://www.home-assistant.io/integrations/zwave/#network-key

Doesn't seem to line up with what I see in my configuration.yaml.  From what I recall when I set up Home Assistant originally it said to leave it blank and it would auto-generate...though I don't see anywhere in the GUI that it tells what it generated.


```
zwave:
  config_path: /config/open-zwave/config
```

Seems like though that link and whatever other info I haven't yet found ought to be added to the documentation for where you find this "key" that I still haven't found.

**Comment by MartinHjelmare** [comment]
Where to find the key depends on the integration you're migrating from and what setup you did when configuring the integration. This will be described in more detail in the respective integration docs and there are also more detailed community guides linked already from the migration docs.

If you're migrating from the Z-Wave integration we recommend using the migration wizard after upgrading to core-2022.3.0 that will be released next week.

**Comment by mmiller7** [comment]
Ah, ok.  I've been dreading the migration (since everything with the legacy zwave is working perfectly) but I've been increasingly worried that I'll get an update and it will be gone leaving me with nothing working and no time to un-break it.

'grep'-ing thru I found something that might be it (looked right) under a hidden folder but trying to put that in the wizard just failed with no error information so that's apparently wrong.

**Comment by mmiller7** [comment]
sadly core-2022.3.5 doesn't seem any better

**Comment by MartinHjelmare** [comment]
Better in what regard?

**Comment by mmiller7** [comment]
Useless errors like "Object Object [OK]" and "Unable to get data [OK]" or similar not covered in documentation, after much searching I finally found a random post that said where to actually find the legacy zwave "key" but still took a TON of trial and error and I'm not even sure if I did it right because there's no info on what format it should be in the wizard when you paste it in.

And when it got done, even several of the entities that the wizard suggested it would migrate I still had to figure out by hand again anyway, as well as some ceiling light+fan dimmers that no longer work with the fan % dimmer sliders with Lovelace cards, wrong units on temperatures, and other things.

I ultimately ended up with nothing usable and had to do a restore from backups because I'm out of time after spending the entire day on this I had non-working HVAC, fans, and door locks and couldn't find information to fix them.  Other things with data reported by the devices that did migrate over don't seem to align anymore with what their manufacturers say they should (e.g. verbose info on lock state, scene button ID data) and a lot of what I was finding for suggestions on "look at events not alarm types" only had some random bits of data while missing other pieces and no documentation for what the translation is or why.

I have to assume the vague pop-ups were errors and part of why things went bad, but I can't know for sure.  I also now see that the "changelog" says the old Z-Wave will be going away soon so I guess I'll have to plan what I'm going to say to my roomates about why things are all broken suddenly that worked so well between now and the next update.

After about 10 hours all I can say is "I'm sure glad I did a backup" and I'm back to square zero.

It would be a lot better if the documentation had clear and up to date information about which information to get from where, what format (showing examples) it should be input, with or without quotes/commas/whitespace/etc, and what each step should look like along the way, how long it should take for things (at one point it seemed hung saying it needed to be "ready" after it had been?), etc.
